wlauto/workloads/msword/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -0,0 +1,376 @@
+
+        // Text formatting
+        clickUiObject(BY_DESC, "Bold", CLASS_TOGGLE_BUTTON);
+        UiObject undoButton = clickUiObject(BY_DESC, "Undo", CLASS_BUTTON);
+        startLogger("format_bold_italic_underline");
+        clickUiObject(BY_DESC, "Bold", CLASS_TOGGLE_BUTTON);
+        clickUiObject(BY_DESC, "Italic", CLASS_TOGGLE_BUTTON);
+        clickUiObject(BY_DESC, "Underline", CLASS_TOGGLE_BUTTON);
+        stopLogger("format_bold_italic_underline");
+        // Font size
+        UiObject fontSize = new UiObject(new UiSelector().resourceId(packageID + "fsComboBoxButton").instance(1));
+        startLogger("format_font_size_menu");
+        fontSize.click();
+        UiScrollable list = new UiScrollable(new UiSelector().resourceId(
+            packageID + "galleryListControl").childSelector(new UiSelector().className("android.widget.ScrollView")));
+        list.scrollIntoView(new UiSelector().textContains("36"));
+        stopLogger("format_font_size_menu");
+        startLogger("format_font_size_action");
+        // Colours
+        clickUiObject(BY_TEXT, "36", CLASS_BUTTON, true);
+        stopLogger("format_font_size_action");
+        startLogger("format_font_colour");
+        clickUiObject(BY_DESC, "Font Colour", CLASS_TOGGLE_BUTTON);
+        clickUiObject(BY_DESC, "Red", true);
+        stopLogger("format_font_colour");
+        startLogger("format_font_highlight");
+        clickUiObject(BY_DESC, "Highlight", CLASS_TOGGLE_BUTTON);
+        clickUiObject(BY_DESC, "Yellow", true);
+        stopLogger("format_font_highlight");
+
+        // Dismiss format menu
+        getUiDevice().pressBack();
+        getUiDevice().waitForIdle();
+        getUiDevice().pressBack();
+
+        // Close file
+        startLogger("toggle_app_menu");
+        clickUiObject(BY_ID, packageID + "Hamburger");
+        stopLogger("toggle_app_menu");
+        startLogger("close_document");
+        clickUiObject(BY_TEXT, "Close", CLASS_BUTTON, true);
+        stopLogger("close_document");
+        deleteDocuments();
+    }
+
+    public void testExistingDocument() throws Exception {
+        openDocument(documentName);
+        // Dismiss tooltip if it appears
+        UiObject tooltip = new UiObject(new UiSelector().textContains("Got it").className(CLASS_BUTTON));
+        startLogger("document_tooltip");
+        if (tooltip.waitForExists(WAIT_TIMEOUT_5SEC)) {
+            tooltip.click();
+        }
+        stopLogger("document_tooltip");
+
+        // Scroll down the document
+        for (int i = 0; i < SCROLL_SWIPE_COUNT; i++) {
+            startLogger("doc_scroll_down_" + i);
+            getUiDevice().pressKeyCode(KeyEvent.KEYCODE_PAGE_DOWN);
+            stopLogger("doc_scroll_down_" + i);
+            // SystemClock.sleep(SCROLL_WAIT_TIME_MS);
+        }
+
+        // show command palette
+        UiObject paletteToggle = new UiObject(new UiSelector().resourceId(packageID + "paletteToggleButton"));
```

Comment:
They used more than once in this code block, and are related, so seemed like a clearer place to declare them. Note that `paletteToggle` is used in the lines immediately following this declaration.